### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by ssaviak**

### CODE REVIEW

**Review by Manankin — COMMENTED** [review]
good job!!

You can use only one line statement 
[this[i], this[i + 1]] = [this[i + 1], this[i]];
instead

**Review by Manankin — APPROVED** [review]

File: src/arrayMethodSort.js [hunk 1]
```diff
@@ -4,8 +4,26 @@
...
+
+    while (isReplaced !== false) {
+      isReplaced = false;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        if (callback(this[i], this[i + 1]) > 0) {
+          const bufferValue = this[i];
+
+          this[i] = this[i + 1];
+          this[i + 1] = bufferValue;
```
Comment (Manankin) [inline src/arrayMethodSort.js:20]: good job!!

You can use only one line statement
[this[i], this[i + 1]] = [this[i + 1], this[i]];
instead

**Review by Manankin — APPROVED** [review]

File: src/arrayMethodSort.js [hunk 2]
```diff
@@ -4,8 +4,26 @@
...
+  const defaultSort = (a, b) => (a.toString() > b.toString() ? 1 : -1);
+
+  [].__proto__.sort2 = function(callback = defaultSort) {
+    let isReplaced = true;
+
+    while (isReplaced !== false) {
+      isReplaced = false;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        if (callback(this[i], this[i + 1]) > 0) {
```
Comment (YegorKorenkov) [inline src/arrayMethodSort.js:16]: You can move this condition in separate variable for make more readable code

**Review by YegorKorenkov — APPROVED** [review]
Good job! Please check minor remarks!

**Pull request closed without merging**